### PR TITLE
optimize: auto load the css of remote's dependencies

### PR DESCRIPTION
**Pull request by Timehello**
#147

### CODE REVIEW

File: packages/lib/src/prod/shared-production.ts [hunk 1]
```diff
@@ -289,16 +289,16 @@ export function prodSharedPlugin(
...
-      )
+      // const exposesFlag = parsedOptions.prodExpose.some((expose) =>
+      //   isSameFilepath(expose[1].id, facadeModuleId as string)
+      // )
       const needSharedImport =
         isRemote &&
         parsedOptions.prodShared.length > 0 &&
         chunk.type === 'chunk' &&
-        (sharedFlag || exposesFlag) &&
+        // (sharedFlag || exposesFlag) &&
```
Comment (ruleeeer) [inline packages/lib/src/prod/shared-production.ts:301]: It is not a good practice to skip the check altogether ,because only the components of exposes need the shard function, other components that are not related to expose should not use shared

Comment (Timehello) [reply packages/lib/src/prod/shared-production.ts]: yes, it should be handled with more logically here. the related codes were reverted

**Comment by ruleeeer** [comment]
thanks for your PR!
